tests/src/FunctionalJavascript/ContactSubmissionTest.php

```diff
@@ -120,8 +124,8 @@ public function testSelectContactElement() {
     $result_debug = var_export($contact_result, TRUE);
 
     $this->assertEquals(1, $contact_result['count'], $result_debug);
-    $this->assertEquals('Frederick', $contact_result['values'][0]['first_name'], $result_debug);
-    $this->assertEquals('Pabst', $contact_result['values'][0]['last_name'], $result_debug);
+    $this->assertEquals($params['first_name'], $contact_result['values'][0]['first_name'], $result_debug);
+    $this->assertEquals($params['last_name'], $contact_result['values'][0]['last_name'], $result_debug);
   }
```

Comment:
And same here -> I do like actual names in the Assert. I find it makes the test more readable.

Reply:
I've pushed a commit to your fork to introduce you to Jann Arden. More Canadian artists to come! 🙂